src/navigation.ts

```diff
@@ -1449,6 +1501,36 @@ function fakeEventForNode(node: Blockly.ASTNode): PointerEvent {
   });
 }
 
+/**
+ * Create a fake PointerEvent for opening the action menu for the
+ * given ASTNode of type NEXT or PREVIOUS.
+ *
+ * For now this just puts the action menu in the same place as the
+ * context menu for the source block.
+ *
+ * @param node The node to open the action menu for.
+ * @returns A synthetic pointerdown PointerEvent.
+ */
+function fakeEventForStackNode(node: Blockly.ASTNode): PointerEvent {
```

Comment:
Why is this `fakeEventForStackNode` and not `fakeEventForConnectionNode`?

Reply:
Because there needs to be an equivalent one for input nodes.  (Maybe the two will end up being folded together, but I didn't want to assume that from the outset.)

Reply:
I'd rather have one method to handle all of the connections, rather than using "stack" (which has had a different meaning in other places in this code).